src/XProfile/VirtoCommerce.XProfile/Schemas/ContactType.cs

```diff
@@ -35,7 +35,12 @@ public ContactType(IOrganizationAggregateRepository organizationAggregateReposit
                 "dynamicProperties",
                 "Contact's dynamic property values",
                 QueryArgumentPresets.GetArgumentForDynamicProperties(),
-                context => dynamicPropertyResolverService.LoadDynamicPropertyValues(context.Source.Contact, context.GetArgumentOrValue<string>("cultureName")));
+                context =>
+                {
+                    context.CopyArgumentsToUserContext();

```

Comment:
Can we extend https://github.com/VirtoCommerce/vc-module-experience-api/blob/dev/src/VirtoCommerce.ExperienceApiModule.Core/Schemas/ExtendableGraphType.cs#L9?

Reply:
Done.